{
  "uid" : "450f043e48c02b1",
  "name" : "loginTest",
  "title" : "Login test",
  "time" : {
    "start" : 1499350500322,
    "stop" : 1499350503906,
    "duration" : 3584
  },
  "summary" : {
    "attachments" : 0,
    "steps" : 0
  },
  "failure" : {
    "message" : "AssertionError: The following asserts failed:\n\t3. Entry is displayed for observer\t3. Entry is displayed for observer expected [false] but found [true]",
    "stackTrace" : "java.lang.AssertionError: The following asserts failed:\n\t3. Entry is displayed for observer\t3. Entry is displayed for observer expected [false] but found [true]\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:48)\n\tat SimpleTest.loginTest(SimpleTest.java:25)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:497)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:639)\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:773)\n\tat org.testng.TestRunner.run(TestRunner.java:623)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:357)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:259)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1110)\n\tat org.testng.TestNG.run(TestNG.java:1018)\n\tat org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)\n\tat org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)\n\tat org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)\n\tat org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)\n\tat org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)\n\tat org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)\n\tat org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)\n"
  },
  "description" : null,
  "severity" : "NORMAL",
  "status" : "FAILED",
  "testId" : {
    "name" : "5.1.2",
    "url" : "5.1.2"
  },
  "suite" : {
    "uid" : "1d12da24971ffcf0",
    "name" : "Surefire suite : Surefire test",
    "title" : "Surefire suite : Surefire test"
  },
  "steps" : [ ],
  "attachments" : [ ],
  "issues" : [ ],
  "labels" : [ {
    "name" : "testId",
    "value" : "5.1.2"
  }, {
    "name" : "feature",
    "value" : "Login test"
  }, {
    "name" : "host",
    "value" : "TYULIKOVV7"
  }, {
    "name" : "thread",
    "value" : "10144@TYULIKOVV7.main(1)"
  }, {
    "name" : "suite-name",
    "value" : "Surefire suite : Surefire test"
  }, {
    "name" : "suite-title",
    "value" : "Surefire suite : Surefire test"
  }, {
    "name" : "framework",
    "value" : "TestNG"
  }, {
    "name" : "language",
    "value" : "JAVA"
  } ],
  "parameters" : [ ]
}